File: lua/refactoring/tests/refactor/106/cpp/class-function/extract.expected.cpp
#include <iostream>

class Coconut {

void foo(int a, int test, int test_other) {
    for (int x = 0; x < test_other + test; x++) {
      std::cout << x << " " << a << std::endl;
    }
}


  void simpleFunction(int a) {
    int test = 1, test_other = 11;
    foo(a, test, test_other);
  }
};

File: lua/refactoring/tests/refactor/106/cs/class-function/extract.expected.cs
class Extract
{

public static void foo(int a, int test, int test_other) {
        for (int x = 0; x < test_other + test; x++)
        {
            Console.WriteLine(x + " " + a);
        }
}


    public static void simpleFunction(int a)
    {
        int test = 1,
            test_other = 11;
            foo(a, test, test_other);
    }
}
